src-v2/Not/Not.Blazor/TM/NotCollapse.razor

```diff
@@ -0,0 +1,24 @@
+﻿
+<MudCollapse  Expanded="_expanded" Class="@(Class + _displayedClass)">
+    @ChildContent
+</MudCollapse>
+
+@code {
+    [Parameter]
+    public RenderFragment ChildContent { get; set; } = default!;
+    [Parameter]
+    public string Class { get; set; } = default!;
+    [Parameter]
+    public bool Expanded { get; set; }
+    [Parameter]
+    public bool Displayed { get; set; }
```

Comment:
What's the idea of having both Displayed and Exapanded?

Reply:
The point of the NotCollapse component is to remove the collapsed element from the document flow i.e stop element from taking screen real estate. Hence the Displayed property enables the removal.